ctsit/send_rx
# Adapting Send Rx to work with User Profile module

This PR depends on the following one:
https://github.com/ctsit/redcap_user_profile/pull/1

Review steps for existing Send Rx environments:
- [ ] Add and enable [User Update](https://github.com/ctsit/redcap_user_profile/tree/develop) module
- [ ] Remove SendRx Sites project
- [ ] Open the [updated README from this PR](https://github.com/tbembersimeao/send_rx/blob/develop/README.md)
- [ ] Follow steps 2, 3 and 4.8 of **Configuration** section
- [ ] Follow step 2 of **Sending your First Test Prescription** section
- [ ] Make sure you are still able to:
 - Grant and revoke staff permissions
 - Set prescriber on patient/prescription form
 - Send prescriptions containing prescribers info

Review steps if starting from scratch:
- [ ] Follow installation and configuration steps from [this README](https://github.com/tbembersimeao/send_rx/blob/develop/README.md)
- [ ] Make sure you are able to:
 - Grant and revoke staff permissions
 - Set prescriber on patient/prescription form
 - Send prescriptions containing prescribers info
